### PR TITLE
CLDR-14613 Modernize, remove USE_DOJO-dependent code

### PR DESCRIPTION
**Pull request by btangmu**
-Deleted many obsolete files; not yet all

-Deleted parts of SurveyAjax.java and SurveyTool.java

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14613
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java [hunk 1]
```diff
@@ -384,80 +209,29 @@ private void includeCss(HttpServletRequest request, PrintWriter out) {
...
         // Per https://en.wikipedia.org/wiki/JQuery#Release_history --
-        // jquery 1.11: January 24, 2014
         // jquery 3.5.1: May 4, 2020
-        final boolean doUseDojo = useDojo(request);
-        if (doUseDojo) {
-            out.write("<script src='//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js'></script>\n");
-        } else {
-            out.write("<script src='//ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js'></script>\n");
-        }
+        out.write("<script src='//ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js'></script>\n");
```
Comment (srl295) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java:221]: yeah… we should fold this into the bundle.

That works easily with jQuery, a little harder with jquery UI for some reason.

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java [hunk 2]
```diff
@@ -384,80 +209,29 @@ private void includeCss(HttpServletRequest request, PrintWriter out) {
...
-            out.write(prefix + "CldrDojoLoad" + js); // CldrDojoLoad.js
-            out.write(prefix + "CldrDojoTable" + js); // CldrDojoTable.js
-            out.write(prefix + "bootstrap.min.js" + tail); // exceptional
-            out.write(prefix + "redesign" + js); // redesign.js
-            out.write(prefix + "review" + js); // review.js
-            out.write(prefix + "CldrDojoGui" + js); // CldrGuiDojo.js
-        } else {
-            out.write(prefix + "bootstrap.min.js" + tail);
-        }
+        // Autosize 4.0.2 (2018-04-30 per changelog.md), see http://www.jacklmoore.com/autosize
```
Comment (srl295) [inline tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java:232]: same. these should be loaded in the bundle.

**Review by srl295 — APPROVED** [review]
hooray for deletion 🎊 

i think we're getting close to being able to have a static `index.html` at the top level.

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java (hunk 2 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java:232]: yes

File: tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java:221]: yes

**Comment by btangmu** [comment]
> hooray for deletion 🎊
> 
> i think we're getting close to being able to have a static `index.html` at the top level.

That will be a pleasant change!

**Comment by btangmu** [comment]
Oops, removal of createAndLogin.jsp conflicts with my other PR in which I just removed one line from the file.

Removing the whole file is correct, since the new back end is in AdminPanel.java,  `org.unicode.cldr.web.AdminPanel.createAndLogin`.

**Comment by btangmu** [comment]
Sorry about the noise, please see https://github.com/unicode-org/cldr/pull/1161

**Pull request closed without merging**